docs/public-networks/concepts/parallel-transaction-execution.md

```diff
@@ -0,0 +1,189 @@
+---
+sidebar_position: 4
+description: Learn about parallel transaction execution.
+tags:
+- public networks
+---
+
+# Parallel transaction execution
+
+Besu supports parallel transaction execution, using an optimistic approach to parallelize
+transactions within a block.
+This page provides an [overview of the mechanism](#parallelization-mechanism-overview), and some key
+[metrics](#metrics) indicating that this feature significantly improves Besu's performance.
+
+:::warning Important
+Parallel transaction execution is an early access feature.
+You can enable it using the `--Xbonsai-parallel-tx-processing-enabled` option.
+:::
+
+## Parallelization mechanism overview
+
+When parallel transaction execution is enabled, Besu initially executes all transactions within a
+block in parallel, operating under the optimistic assumption that they can all be executed
+concurrently without conflict.
+This parallel execution runs in the background, and Besu proceeds to sequentially process the
+transactions without waiting for the parallel execution to complete.
+
+The following flowchart outlines the sequential processing flow:
```

Comment:
I was initially confused by the diagram because I was trying to see where the parallel execution happens. Is it possible to show the parallel execution that happens in the background as well?